src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let result = days * 40;
```

Comment:
See checklist, we don't want magic numbers, use variables :) also try to use return in `if`